pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -345,6 +347,8 @@ func (c Client) WaitForVulnzAnalysis(containerImage string, timeout time.Duratio
 			}
 			switch updated.GetDiscovered().GetDiscovered().GetAnalysisStatus() {
 			case discovery.Discovered_FINISHED_SUCCESS:
+				elapsed := time.Since(start)
+				glog.Infof("Find vulnerabilities after %s", elapsed)
```

Comment:
Find -> Found
(if that's what you meant)

Reply:
ack.